### PR TITLE
LPS-147203 | Add Autom. test ClientExtension#CannotConfigureWorkflowSiteLevel

### PR DESCRIPTION
**Pull request by abhnerramos**
- My ticket

https://issues.liferay.com/browse/LPS-147203

- Description

Add Automation for test ClientExtension#CannotConfigureWorkflowSiteLevel

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase [hunk 1]
```diff
@@ -99,9 +99,12 @@ definition {
...
 	@priority = "3"
 	test CannotConfigureWorkflowSiteLevel {
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			portlet = "Workflow",
+			site = "${siteName}");
 
-		// TODO LPS-171621 CannotConfigureWorkflowSiteLevel pending implementation
-
+		AssertTextNotPresent(value1 = "Client Extension");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase]: Add a search for text `Client Extension`.

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase]: cc/ @mateusralv can you help show how to wrap the steps with `task( )` ?

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:client-extensions:nocompile

**Comment by john-co** [comment]
Looks good to me! :sunglasses:

If you see the test pass on ci:test:client-extensions, let's go ahead and run `ci:forward`

For the next tests, please send the PR to user `liferay-frontend` so I can help review. Thanks!

**Comment by abhnerramos** [comment]
> Looks good to me! sunglasses
> 
> If you see the test pass on ci:test:client-extensions, let's go ahead and run `ci:forward`
> 
> For the next tests, please send the PR to user `liferay-frontend` so I can help review. Thanks!

Perfect! Thank you @john-co

**Comment by john-co** [comment]
ci:test:frontend-client-extensions:nocompile

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
@mateusralv , please check the frontend-client-extensions. It seems like it's missing all the client extension tests. Not sure if it's related to the test suite name change.

**Comment by mateusralv** [comment]
The test is [passing](https://testray.liferay.com/home/-/testray/case_results/2189089408) the ` ci:test:frontend` but does not show in `ci:test:frontend-client-extensions`, the routine does not seem to work as we expected, I will take a look at it!

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**